figure_generation/manuscript_figure_generation.py

```diff
@@ -0,0 +1,566 @@
+)
+
+panel_two_size = (
+    float(panel_two.root.attrib['width']),
+    float(panel_two.root.attrib['height'])
+)
+
+scale_x = 0.45
+scale_y = 0.45
+
+print(f"original: {panel_two_size}")
+print(f"scaled:{(panel_two_size[0]*scale_x, panel_two_size[1]*scale_y)}")
+
+panel_two = panel_two.getroot()
+panel_two.scale_xy(x=scale_x, y=scale_y)
+panel_two.moveto(620,20)
+
+
+# In[14]:
+
+
+panel_three = (
+    sg.fromfile(f"{str(pca_association_path)}/scatterplot_files/pca01_v_pca02.svg")
+)
+
+
+panel_three_size = (
+    np.round(float(panel_three.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_three.root.attrib['height'][:-2])*1.33, 0)
+)
+
+scale_x = 1
+scale_y = 1
+
+print(f"original: {panel_three_size}")
+print(f"scaled:{(panel_three_size[0]*scale_x, panel_three_size[1]*scale_y)}")
+
+panel_three = panel_three.getroot()
+panel_three.scale_xy(x=scale_x, y=scale_y)
+panel_three.moveto(10,430)
+
+
+# In[15]:
+
+
+panel_four = (
+    sg.fromfile(f"{str(pca_association_path)}/category_box_plot/category_box_plot_pc1.svg")
+)
+
+
+panel_four_size = (
+    np.round(float(panel_four.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_four.root.attrib['height'][:-2])*1.33, 0)
+)
+
+scale_x = 1
+scale_y = 1
+
+print(f"original: {panel_four_size}")
+print(f"scaled:{(panel_four_size[0]*scale_x, panel_four_size[1]*scale_y)}")
+
+panel_four = panel_four.getroot()
+panel_four.scale_xy(x=scale_x, y=scale_y)
+panel_four.moveto(630,430)
+
+
+# In[16]:
+
+
+panel_five = (
+    sg.fromfile(f"{str(pca_association_path)}/category_box_plot/category_box_plot_pc2.svg")
+)
+
+
+panel_five_size = (
+    np.round(float(panel_five.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_five.root.attrib['height'][:-2])*1.33, 0)
+)
+
+scale_x = 1
+scale_y = 1
+
+print(f"original: {panel_five_size}")
+print(f"scaled:{(panel_five_size[0]*scale_x, panel_five_size[1]*scale_y)}")
+
+panel_five = panel_five.getroot()
+panel_five.scale_xy(x=scale_x, y=scale_y)
+panel_five.moveto(20,800)
+
+
+# In[17]:
+
+
+panel_one_label = sg.TextElement(10, 20, "A", size=22, weight="bold")
+panel_one_image_label = sg.TextElement(200, 385, "PCA 1", size=22, weight="bold")
+panel_two_label = sg.TextElement(600, 20, "B", size=22, weight="bold")
+panel_two_image_label = sg.TextElement(820, 385, "PCA 2", size=22, weight="bold")
+panel_three_label = sg.TextElement(10, 420, "C", size=22, weight="bold")
+panel_four_label = sg.TextElement(620, 420, "D", size=22, weight="bold")
+panel_five_label = sg.TextElement(10, 800, "E", size=22, weight="bold")
+
+
+# In[18]:
+
+
+figure_two = sg.SVGFigure("1600", "1200")
+figure_two.append([
+    panel_one,
+    panel_two,
+    panel_three,
+    panel_four,
+    panel_five,
+    panel_one_label,
+    panel_one_image_label,
+    panel_two_label,
+    panel_two_image_label,
+    panel_three_label,
+    panel_four_label,
+    panel_five_label
+])
+display(SVG(figure_two.to_str()))
```

Comment:
The word plots look super good! They're very legible even in a small space.

It's difficult to read the words on the other figures though due to their size. If they look good on your side then it's probably just a side-effect of me using nbviewer.

Minor layout note, if you swap C and E or drop D then the similar plots will be next to each other

Reply:
(Fig. 4)

Reply:
Followup on the layout..I feel like A,B and C are one panel since B and C refer to the PC patterns. I wonder if you could make B and C smaller and remove the panel label so its all panel A

And then I would drop D so that D and E are on the same level as Ben mentioned.